app/assets/images/administrate/search.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
+<svg id="search" width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@tysongach no, it is not for styling.

I'm not sure if you had the opportunity to run administrate with these changes, but since I'm replacing `inline_svg` with `svg`/`use` tags this ID is required by the `use` tag to select a node or group of nodes to be duplicated into the HTML DOM https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use

The following is the example on how the search icon will look in the DOM.
```
<span class="search__icon">
  <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <use xlink:href="http://localhost:7003/assets/administrate/search-360c782d91a21edd53f5754c3fd8f8984ec5d9b36389f6df1fe618a4e50405b6.svg#search" width="16" height="16">
    </use>
  </svg>
</span>
``` 

As you can see in the attribute `xlink:href` of `use` tag, it does contains the URL of the sag file but also the ID of the nodes to duplicate. `use` tag requires this ID.